doc/source/conf.py

```diff
@@ -18,10 +18,7 @@
 #
 import os
 import sys
-# Note that we need a specific version of sphinx_rtd_theme. This can be obtained with:
```

Comment:
No longer needed because it's set in the requirements file. That's a good change, and makes python environment requirements easier to see.